voorhoede/voorhoede-website
# Switch /faq to landing page

## What changes were made
- Delete FAQ page
- Add links field to Footer model
- Update Dato menu

Differences with original FAQ:
- I opted for centering the text because left aligned it's very long and harder to read. 
- I'm using /h3 on the content because /h2 look super big.

## How to test or check results
Visit the new page: https://deploy-preview-709--voorhoede-website.netlify.app/en/faq/
Check that the footer links work as expected.

## Checks
